_data/simple-icons.json

```diff
@@ -6802,6 +6802,11 @@
             "hex": "556DB3",
             "source": "https://www.pcgamingwiki.com/wiki/Home"
         },
+        {
+            "title": "Peak Design",
+            "hex": "1C1B1C",
+            "source": "https://cdn.shopify.com/s/files/1/2986/1172/t/6/assets/headerlogo-10.svg"
```

Comment:
Since the SVG can be fond in their website header, you can use https://www.peakdesign.com/ only as source

Reply:
@service-paradis I've updated the source now 🙂